src/OpenTK/Math/Vector2.cs

```diff
@@ -667,6 +693,19 @@ public static void BaryCentric(ref Vector2 a, ref Vector2 b, ref Vector2 c, floa
         }
 
         /// <summary>
+        /// Transforms a vector by a Matrix rotation.
+        /// </summary>
+        /// <param name="vec">The vector to transform.</param>
+        /// <param name="mat">The matrix to rotate the vector by.</param>
+        /// <returns>The result of the operation.</returns>
+        public static Vector2 Transform(ref Matrix2 mat, ref Vector2 vec)
```

Comment:
sure about this? the same methods already exist in opentk for vector3, so i'd say having this here makes sense?